guides/pages/how-to-filter-threads-with-the-rest-api.mdx

```diff
@@ -0,0 +1,67 @@
+---
+meta:
+  title: "How to filter threads with the REST API"
+  description:
+    "Learn how to filter for certain threads using their metadata with our REST
+    API"
+---
+
+When using Comments and retrieving threads with our REST API, it’s possible to
+filter for specific threads using
+[custom metadata](/docs/products/comments/metadata) and our custom query
+language. This enables the
+[Get Threads REST API](/docs/api-reference/rest-api-endpoints#get-rooms-roomId-threads)
+to have filtering that works the same as with
+[`useThreads`](/docs/api-reference/liveblocks-react#useThreads) and
+[`liveblocks.getThreads`](/docs/api-reference/liveblocks-node#get-rooms-roomId-threads).
+
+## Query language
+
+You can filter threads by their metadata, allowing you to select for certain
+properties, values, or even for string prefixes. Filters can be combined
+together using `AND` logic.
+
+### Cheatsheet
+
+```js
+// Threads with the `{ status }` metadata
+metadata['status']
+
+// Threads with both `{ status }` and `{ priority }` metadata
+metadata['status'] AND metadata['priority']
```

Comment:
Same here. A valid example would be:

```suggestion
// Threads with both `{ status }` and `{ priority }` metadata
metadata['status']:'open' AND metadata['priority']>=3
```

Reply:
For now we do not support `>=` only `:` (equal operator)

Reply:
Wait, `metadata` is [declared as mixed](https://github.com/liveblocks/liveblocks-backend/pull/331/files#diff-a616200d88dd4f8e1a8824a81d048bac08d0af283f1dde72d50bd4f0bd872e69R8), right? And mixed [supports](https://github.com/liveblocks/liveblocks-cloudflare/blob/36137bd6d838f64a62d00437b0877e9aa443e6dc/packages/liveblocks-query-parser/README.md?plain=1#L45) `>=` operators when used with numeric values. The language would definitely accept that query. Are you saying the parser will, but then at query time this would be an error?

Reply:
Oops had missed this comment: the parser accepts the value but I have validation code after which rejects the operator in the backend